src/components/ProgressBar/index.jsx

```diff
@@ -0,0 +1,87 @@
+import styled, { css } from 'styled-components';
+import { sizes, zIndexes, colors } from '../../styles/variables';
+
+export const Bar = styled.span`
```

Comment:
Maybe you can have a specific file to the `<Bar />` component, and another file to the `<ProgressBarStyled />`